### ISSUE TITLE
Granted Registries do not support Gerrit repositories

### ISSUE THREAD

**Issue by sosheskaz** [common-fate/granted#310]
Granted parses the repository string to look for organization name, but not all git servers have organization names, such as [Gerrit](https://www.gerritcodereview.com/).

Gerrit repositories generally exist in the format `https://gerrit.googlesource.com/gerrit`. This reliably produces the following error:

```bash
$ granted --verbose registry add https://gerrit.googlesource.com/gerrit
[DEBUG] parsing the provided url to get host, organization and repo name for https://gerrit.googlesource.com/gerrit
[✘] unable to parse the provided git url 'https://gerrit.googlesource.com/gerrit'
```

**Comment by sosheskaz** [comment]
Some git webservers may also have unexpected formats, ex. if they are running on a server that does path-pased forwarding, ex: `https://my-website.com/gitea/organization/repo`. I do not know how well it would handle that case.

**Comment by JoshuaWilkes** [comment]
@sosheskaz I have continued the conversation on https://github.com/common-fate/rfds/discussions/2 with a proposal for how we can improve the support for non Github URLs, if you have any thoughts or feedback on the proposal please feel free to join the discussion there.

**Comment by Eddie023** [comment]
Should be fixed with #317. Closing the issue.
